db/migrate_cards/20180105134605_created_at_for_answers.rb

```diff
@@ -0,0 +1,19 @@
+# -*- encoding : utf-8 -*-
+
+class CreatedAtForAnswers < Card::Migration
+  def up
+    Answer.where(created_at: nil).each do |a|
+      next unless a.card
+      if a.card.type_id == Card::MetricValueID
+        if a.card.calculated?
+          a.update_attributes! created_at: a.card.created_at, answer_id: nil
+          a.card.delete!
```

Comment:
I think this a.card.delete! is safe, because the event that deletes the lookup record uses answer_id (and that is now nil).